pathspider/cmd/upload.py

```diff
@@ -0,0 +1,284 @@
+
+import pycurl
+import argparse
+import logging
+import json
+import bz2
+import dateutil.parser
+import os
+
+from io import BytesIO
+
+BASELINK = "https://v3.pto.mami-project.eu/raw/"
+TOKEN = "APIKEY"
```

Comment:
This should not have a default. The command should just refuse to run if there's no token provided on the command line.